docs/docsite/rst/scenario_guides/guide_online.rst

```diff
@@ -0,0 +1,29 @@
+*************************
+Using Ansible with Online
+*************************
+
+Introduction
+============
+
+Online is a French hosting company mainly known for providing baremetal servers named Dedibox.
+Check it out: https://www.online.net/en
+
+Dynamic inventory for Online resources
+--------------------------------------
+
+Ansible has a dynamic inventory plugin that can list your resources.
+
+1. Create a YAML configuration such as ``online_inventory.yml``
```

Comment:
If the YAML codeblock below should be the content of the configuration file, add "with this content:" to this line for clarity.